apps/web/src/app/(prompt)/edit/[agentId]/[postGroupId]/detail/_components/EditSidebar/EditSidebar.tsx

```diff
@@ -93,8 +94,13 @@ function EditSidebarContent() {
       confirmButton: '삭제하기',
       cancelButton: '취소',
       confirmButtonProps: {
-        onClick: () => {
-          deletePost(postId);
+        onClick: async () => {
+          try {
+            await deletePost(postId);
+            router.push(ROUTES.CREATE);
+          } catch (error) {
+            toast.error('삭제하지 못했어요.');
+          }
```

Comment:
```suggestion
        onClick: async () => {
          try {
            await deletePost(postId);
            router.push(ROUTES.CREATE);
          } catch (error) {
            toast.error('삭제하지 못했어요.');
          }
```

p1: 여기도 마찬가지입니다! 그리고 mutateAsync가 아니라 mutate 사용해도 문제 없을 듯 해요.

Reply:
https://github.com/YAPP-Github/25th-Web-Team-1-FE/pull/135#discussion_r1960431171

위 처럼 콜백으로 처리 부탁드려요

Reply:
반영했습니다